src/Microsoft.Data/DataFrame.BinaryOperations.cs

```diff
@@ -332,7 +326,7 @@ public DataFrame RightShift(int value, bool inPlace = false)
             }
             return retDataFrame;
         }
-        public DataFrame Equals<T>(IReadOnlyList<T> values)
+        public DataFrame PairwiseEquals<T>(IReadOnlyList<T> values)
```

Comment:
Does `Elementwise` sound better than `Pairwise`?

Reply:
I think so.... but honestly, I'm not super excited about either. `Elementwise` is probably best for now.

---
In reply to: [335084385](https://github.com/dotnet/corefxlab/pull/2750#discussion_r335084385) [](ancestors = 335084385)